apps/api/src/music/music.controller.ts

```diff
@@ -32,44 +34,109 @@ export class MusicController {
 
   @Post()
   @ApiOperation({ description: 'Create music' })
+  async create(@Body() data: CreateMusicDto) {
+    const result = await this.musicService.create({
+      ...data,
+    });
+
+    this.logger.debug(`MUSIC CREATE | Recieved data client:\n${result}`);
+    return {
+      message: 'Music was successfully created',
```

Comment:
it would better if we return a message key and translate it on FE based on user's preferred language .
```suggestion
      messageKey: 'MUSIC_CREATED',
```

Reply:
LGTM, maybe as Issue

Reply:
This idea can be followed in #18 from now on.